<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="1" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-01-19T18:10:36 IST" name="Api Demo Suite" finished-at="2024-01-19T18:12:11 IST" duration-ms="94990">
    <groups>
    </groups>
    <test started-at="2024-01-19T18:10:36 IST" name="Wifi Settings Test Case" finished-at="2024-01-19T18:11:43 IST" duration-ms="66833">
      <class name="com.appium.test.WifiSettingsTest">
        <test-method is-config="true" signature="SetUp()[pri:0, instance:com.appium.test.WifiSettingsTest@50ecde95]" started-at="2024-01-19T18:10:36 IST" name="SetUp" finished-at="2024-01-19T18:10:54 IST" duration-ms="17531" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method signature="WifiSettings(java.util.Hashtable)[pri:0, instance:com.appium.test.WifiSettingsTest@50ecde95]" started-at="2024-01-19T18:10:54 IST" name="WifiSettings" data-provider="DataWifiSettings" finished-at="2024-01-19T18:11:01 IST" duration-ms="6960" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{wifiname=Kaushik Mukherjee}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[OK text is not found expected [HI] but found [OK]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: OK text is not found expected [HI] but found [OK]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at com.appium.utilities.Utilities.AssertTextAndroid(Utilities.java:51)
at com.appium.test.WifiSettingsTest.WifiSettings(WifiSettingsTest.java:223)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- WifiSettings -->
        <test-method is-config="true" signature="TearDownMethod(org.testng.ITestResult)[pri:0, instance:com.appium.test.WifiSettingsTest@50ecde95]" started-at="2024-01-19T18:11:01 IST" name="TearDownMethod" finished-at="2024-01-19T18:11:02 IST" duration-ms="1276" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=WifiSettings status=FAILURE method=WifiSettingsTest.WifiSettings(java.util.Hashtable)[pri:0, instance:com.appium.test.WifiSettingsTest@50ecde95] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDownMethod -->
        <test-method is-config="true" signature="SetUp()[pri:0, instance:com.appium.test.WifiSettingsTest@50ecde95]" started-at="2024-01-19T18:11:02 IST" name="SetUp" finished-at="2024-01-19T18:11:15 IST" duration-ms="12994" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method signature="WifiSettings(java.util.Hashtable)[pri:0, instance:com.appium.test.WifiSettingsTest@50ecde95]" started-at="2024-01-19T18:11:15 IST" name="WifiSettings" data-provider="DataWifiSettings" finished-at="2024-01-19T18:11:22 IST" duration-ms="6847" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{wifiname=Sam Mendes}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[OK text is not found expected [HI] but found [OK]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: OK text is not found expected [HI] but found [OK]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at com.appium.utilities.Utilities.AssertTextAndroid(Utilities.java:51)
at com.appium.test.WifiSettingsTest.WifiSettings(WifiSettingsTest.java:223)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- WifiSettings -->
        <test-method is-config="true" signature="TearDownMethod(org.testng.ITestResult)[pri:0, instance:com.appium.test.WifiSettingsTest@50ecde95]" started-at="2024-01-19T18:11:22 IST" name="TearDownMethod" finished-at="2024-01-19T18:11:23 IST" duration-ms="732" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=WifiSettings status=FAILURE method=WifiSettingsTest.WifiSettings(java.util.Hashtable)[pri:0, instance:com.appium.test.WifiSettingsTest@50ecde95] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDownMethod -->
        <test-method is-config="true" signature="SetUp()[pri:0, instance:com.appium.test.WifiSettingsTest@50ecde95]" started-at="2024-01-19T18:11:23 IST" name="SetUp" finished-at="2024-01-19T18:11:35 IST" duration-ms="12502" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method signature="WifiSettings(java.util.Hashtable)[pri:0, instance:com.appium.test.WifiSettingsTest@50ecde95]" started-at="2024-01-19T18:11:35 IST" name="WifiSettings" data-provider="DataWifiSettings" finished-at="2024-01-19T18:11:42 IST" duration-ms="6447" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{wifiname=Peter M}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[OK text is not found expected [HI] but found [OK]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: OK text is not found expected [HI] but found [OK]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at com.appium.utilities.Utilities.AssertTextAndroid(Utilities.java:51)
at com.appium.test.WifiSettingsTest.WifiSettings(WifiSettingsTest.java:223)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- WifiSettings -->
        <test-method is-config="true" signature="TearDownMethod(org.testng.ITestResult)[pri:0, instance:com.appium.test.WifiSettingsTest@50ecde95]" started-at="2024-01-19T18:11:42 IST" name="TearDownMethod" finished-at="2024-01-19T18:11:43 IST" duration-ms="957" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=WifiSettings status=FAILURE method=WifiSettingsTest.WifiSettings(java.util.Hashtable)[pri:0, instance:com.appium.test.WifiSettingsTest@50ecde95] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDownMethod -->
      </class> <!-- com.appium.test.WifiSettingsTest -->
    </test> <!-- Wifi Settings Test Case -->
    <test started-at="2024-01-19T18:11:43 IST" name="Date Widget Test Case" finished-at="2024-01-19T18:12:11 IST" duration-ms="28119">
      <class name="com.appium.test.WifiSettingsTest">
        <test-method is-config="true" signature="SuiteSetUp()[pri:0, instance:com.appium.test.WifiSettingsTest@50ecde95]" started-at="2024-01-19T18:10:26 IST" name="SuiteSetUp" finished-at="2024-01-19T18:10:36 IST" duration-ms="10023" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SuiteSetUp -->
        <test-method is-config="true" signature="SuiteTearDown()[pri:0, instance:com.appium.test.WifiSettingsTest@50ecde95]" started-at="2024-01-19T18:12:11 IST" name="SuiteTearDown" finished-at="2024-01-19T18:12:11 IST" duration-ms="23" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SuiteTearDown -->
      </class> <!-- com.appium.test.WifiSettingsTest -->
      <class name="com.appium.test.DataWidgetsPageActivityPackageTest">
        <test-method is-config="true" signature="SetUp()[pri:0, instance:com.appium.test.DataWidgetsPageActivityPackageTest@37052337]" started-at="2024-01-19T18:11:43 IST" name="SetUp" finished-at="2024-01-19T18:11:55 IST" duration-ms="12659" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method signature="DateWidgetPageActivityPackage()[pri:0, instance:com.appium.test.DataWidgetsPageActivityPackageTest@37052337]" started-at="2024-01-19T18:11:55 IST" name="DateWidgetPageActivityPackage" finished-at="2024-01-19T18:12:10 IST" duration-ms="15066" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DateWidgetPageActivityPackage -->
        <test-method is-config="true" signature="TearDownMethod(org.testng.ITestResult)[pri:0, instance:com.appium.test.DataWidgetsPageActivityPackageTest@37052337]" started-at="2024-01-19T18:12:10 IST" name="TearDownMethod" finished-at="2024-01-19T18:12:11 IST" duration-ms="392" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=DateWidgetPageActivityPackage status=SUCCESS method=DataWidgetsPageActivityPackageTest.DateWidgetPageActivityPackage()[pri:0, instance:com.appium.test.DataWidgetsPageActivityPackageTest@37052337] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDownMethod -->
      </class> <!-- com.appium.test.DataWidgetsPageActivityPackageTest -->
    </test> <!-- Date Widget Test Case -->
  </suite> <!-- Api Demo Suite -->
</testng-results>
